bug: crewAI tests fails and instruction does not work

```bash
___________________________________________ CrewAITest.test_agent_runs ___________________________________________

self = <crewai_agent.test_crewai.CrewAITest testMethod=test_agent_runs>

    def test_agent_runs(self) -> None:
        agents_yaml = parse_yaml(os.path.join(os.path.dirname(__file__), "agents.yaml"))
        workflow_yaml = parse_yaml(
            os.path.join(os.path.dirname(__file__), "workflow.yaml")
        )

        try:
            workflow = Workflow(agents_yaml, workflow_yaml[0])
        except Exception as excep:
            raise RuntimeError("Unable to create agents") from excep

>       result = asyncio.run(workflow.run())
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^

tests/agents/crewai_agent/test_crewai.py:31:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../.pyenv/versions/3.12.8/lib/python3.12/asyncio/runners.py:194: in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
../../.pyenv/versions/3.12.8/lib/python3.12/asyncio/runners.py:118: in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../.pyenv/versions/3.12.8/lib/python3.12/asyncio/base_events.py:686: in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
src/maestro/workflow.py:59: in run
    self._create_or_restore_agents()
src/maestro/workflow.py:95: in _create_or_restore_agents
    agent_instance = cls(agent_def)
                     ^^^^^^^^^^^^^^
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <maestro.agents.crewai_agent.CrewAIAgent object at 0x116c45310>
agent = {'apiVersion': 'maestro/v1alpha1', 'kind': 'Agent', 'metadata': {'labels': {'app': 'crewaitest', 'class': 'DummyCrew',...': {'description': 'test', 'framework': 'crewai', 'instructions': "input = 'Mock agent:'", 'model': 'llama3.1:latest'}}

    def __init__(self, agent: dict) -> None:
        """
        Initializes the workflow for the specified agent.
        The executable code must be within $PYTHONPATH.

        Args:
            agent (dict): Agent Configuration

        Raises:
            RuntimeError: If the 'crewai' library is not installed or failed to import.
            Exception: If the agent configuration is invalid.
        """
        if not enabled:
>           raise RuntimeError(
                f"Cannot initialize CrewAIAgent '{agent.get('metadata', {}).get('name', 'Unknown')}': "
                f"CrewAI support is disabled because the 'crewai' library could not be imported. "
                "To enable, run `pip install maestro[crewai]`"
                f"Original error: {CREWAI_IMPORT_ERROR}"
            )
E           RuntimeError: Cannot initialize CrewAIAgent 'DummyCrew': CrewAI support is disabled because the 'crewai' library could not be imported. To enable, run `pip install maestro[crewai]`Original error: No module named 'crewai'

src/maestro/agents/crewai_agent.py:53: RuntimeError
================================================ warnings summary ================================================
.venv/lib/python3.12/site-packages/pydantic/_internal/_config.py:323
  /Users/maximilien/github/maestro/.venv/lib/python3.12/site-packages/pydantic/_internal/_config.py:323: PydanticDeprecatedSince20: Support for class-based `config` is deprecated, use ConfigDict instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.11/migration/
    warnings.warn(DEPRECATION_MESSAGE, DeprecationWarning)

.venv/lib/python3.12/site-packages/litellm/caching/llm_caching_handler.py:17
  /Users/maximilien/github/maestro/.venv/lib/python3.12/site-packages/litellm/caching/llm_caching_handler.py:17: DeprecationWarning: There is no current event loop
    event_loop = asyncio.get_event_loop()

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
============================================ short test summary info =============================================
FAILED tests/agents/crewai_agent/test_crewai.py::CrewAITest::test_agent_runs - RuntimeError: Cannot initialize CrewAIAgent 'DummyCrew': CrewAI support is disabled because the 'crewai' library could not be imported. To enable, run `pip install maestro[crewai]`Original error: No module named 'crewai'
============================== 1 failed, 65 passed, 2 skipped, 2 warnings in 47.38s ==============================
(maestro) ➜  maestro git:(main) ✗ uv pip install maestro[crewai]
zsh: no matches found: maestro[crewai]
(maestro) ➜  maestro git:(main) ✗ clspip install maestro[crewai]
zsh: no matches found: maestro[crewai]
(maestro) ➜  maestro git:(main) ✗ pip install maestro[crewai]
zsh: no matches found: maestro[crewai]
(maestro) ➜  maestro git:(main) ✗ pip install maestro[crewai]
```

--- a/src/maestro/agents/crewai_agent.py
+++ b/src/maestro/agents/crewai_agent.py
@@ -53,7 +53,7 @@ class CrewAIAgent(BeeAgent):
             raise RuntimeError(
                 f"Cannot initialize CrewAIAgent '{agent.get('metadata', {}).get('name', 'Unknown')}': "
                 f"CrewAI support is disabled because the 'crewai' library could not be imported. "
-                "To enable, run `pip install maestro[crewai]`"
+                "To enable, run `uv sync --all-extras`"
                 f"Original error: {CREWAI_IMPORT_ERROR}"
             )
 

